crates/matrix-sdk-base/Cargo.toml

```diff
@@ -47,6 +47,9 @@ thiserror = { workspace = true }
 tracing = { workspace = true }
 zeroize = { workspace = true, features = ["zeroize_derive"] }
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+tokio = { version = "1.24.2", default-features = false, features = ["sync"] }
```

Comment:
I wonder if it's controversial to add tokio as a base-sdk dependency, best check with the rust team

Reply:
@jplatte could this be a problem?